torchaudio/functional/functional.py

```diff
@@ -1798,3 +1798,23 @@ def rnnt_loss(
         return costs.sum()
 
     return costs
+
+
+def _get_mat_trace(input: torch.Tensor, dim1: int = -1, dim2: int = -2) -> torch.Tensor:
```

Comment:
This function seems not used in `functional`. In that case, isn't `transforms.py` a better place to have?

Reply:
Sure. When PyTorch supports this method, it will be removed from torchaudio anyway. I will move it to transforms then.